src/map.rs

```diff
@@ -239,9 +249,19 @@ where
 
 #[cfg_attr(feature = "inline-more", inline)]
 pub(crate) fn make_hash<K: Hash + ?Sized>(hash_builder: &impl BuildHasher, val: &K) -> u64 {
-    let mut state = hash_builder.build_hasher();
-    val.hash(&mut state);
-    state.finish()
+    #[cfg(feature = "ahash")]
```

Comment:
A comment explaining what is going on here would be helpful (specialization to improve performance).

Reply:
Done